addons/stock/models/stock_scrap.py

```diff
@@ -171,6 +171,8 @@ def action_get_stock_move_lines(self):
 
     def action_validate(self):
         self.ensure_one()
+        if self.scrap_qty <= 0:
+            raise UserError(_('You can not enter negative quantities.'))
```

Comment:
`<=` does not mean negative, it means non-positive.
However "non-positive" is a negation and it's easier to read things without it.
So better message would be `You can only enter positive quantities.`
Additionally it would be better to explicitly state which field is wrong.

Reply:
@albertopoljak Thanks for the suggestions, updated the PR with the changes